caom2utils/caom2utils/tests/test_fits2caom2.py

```diff
@@ -841,7 +940,7 @@ def _get_obs(from_xml_string):
           <caom2:productType>thumbnail</caom2:productType>
           <caom2:releaseType>data</caom2:releaseType>
           <caom2:contentType>text/plain</caom2:contentType>
-          <caom2:contentLength>2606</caom2:contentLength>
+          <caom2:contentLength>2832</caom2:contentLength>
```

Comment:
Has this changed?

Reply:
Yes, it's the help.txt file, and new parameters have been added.